db/version_set.h

```diff
@@ -1023,6 +1026,7 @@ class VersionSet {
   Env* const env_;
   const std::string dbname_;
   const ImmutableDBOptions* const db_options_;
+  MutableDBOptions* mutable_db_options_;
```

Comment:
We can change `mutable_db_options_` dynamically. What happens if `stats_persist_period_sec` is zero when `Recover` is called, but changed to a non-zero value later? When will the `_persistent_stats` column family be created?

Reply:
good question, right now in SetOptions we only check if RepeatableThread needs to be restarted. I guess we need to add a helper function to handle the case where the column family is not created

Reply:
Maybe we should always create this column family. What do you think?